website/docs/guide/ip-address.md

```diff
@@ -77,8 +77,10 @@ E.g.:
 
 ```go
 e.IPExtractor = echo.ExtractIPFromXFFHeader(
-	TrustLinkLocal(false),
-	TrustIPRanges(lbIPRange),
+     echo.TrustLoopback(false), // e.g. ipv4 start with 127. 
+     echo.TrustLinkLocal(false), // e.g. ipv4 start with 169.254
+     echo.TrustPrivateNet(false), // e.g. ipv4 start with 10. or 192.168
+     echo.TrustIPRange(echo.lbIPRange),
```

Comment:
`echo.lbIPRange` should probably `lbIPRange`. Other than that - this PR seems ok

Reply:
done.
thank you